All_materials/20230822_Web_Scraping_Financial_Ombudsman_Service/output_table.md

```diff
@@ -0,0 +1,26 @@
+|    | Ref         | Date                | Bank                                        | Scale of Lost   | Outcome   | CRM   | Sample                                                                                                                                                                                                                                                                                                                                                                                                           |
+|---:|:------------|:--------------------|:--------------------------------------------|:----------------|:----------|:------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|  0 | DRN-4009518 | 2023-06-01 00:00:00 | Lloyds Bank PLC                             | 10^5            | Notupheld | False | omplaint . I know disappointing , I � explain . Was Mr D victim scam ? It dispute Mr D victim scam � initially persuaded invest dating app , introduced third parties applied pressure Mr D part money order access supposed profits . The scammers cut contact Mr D would part money . Did Mr D authorise payments ? In line Payment Services Regulations 2017 ( PSRs ) , Mr D � liable payments � authorise ,  |
```

Comment:
Again, is this supposed to say 'complaint'?

Reply:
yes, buts it the way it been cropped